TutorLizard.BusinessLogic/Data/User.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
If we just want a json file to be present, I'd change the content to just `[]`.
You could set this file to be copied along with the app during compilation. Right click the file in Solution explorer -> Properties -> Copy to output directory -> Always copy.